week-1-ts-ecosystem/w01e01-typescript-basics/src/types/user.ts

```diff
@@ -0,0 +1,8 @@
+import type {IAddress} from "./address";
+
+export interface IUser {
+	name: string,
+	age: number,
+	email: string,
+	address: IAddress
+}
```

Comment:
Na razie za równo `IUser` jak i `IAddress` mogą mieszkać w tym samym pliku
dlatego że nigdzie nie korzystamy z `IAddress` i tylko `IUser` byłby wyeksportowany

Generalnie nie jestem zwolennikiem stosowania notacji `I` do `inteface'ów` w TypeScript. 
Nie widzę z tego wymiernej krzyści.

Te elementy to właśnie Twój `model/` danych i tak mógłby się nazywać folder.